src/autocomplete/util.js

```diff
@@ -54,3 +54,15 @@ exports.getCompletionPrefix = function (editor) {
     }.bind(this));
     return prefix || this.retrievePrecedingIdentifier(line, pos.column);
 };
+
+exports.triggerAutocomplete = function (editor) {
```

Comment:
does this work only with ASCII characters? Should we expect it to work with others as well?